geometry_msgs/Point Point
std_msgs/Float64 size
std_msgs/Float64MultiArray colour